CONTRIBUTING.md

```diff
@@ -0,0 +1,374 @@
+The body should include the motivation for the change and contrast this with previous behavior.
+
+It should contain any information about **Breaking Changes** and is also the place to reference GitHub issues that this commit **Closes**.
+
+**Breaking Changes** should start with the word `BREAKING CHANGE:` with a space or two newlines.
+The rest of the commit message is then used for this.
+
+#### Examples
+
+```commit
+fix(pencil): stop graphite breaking when too much pressure applied
+```
+
+```commit
+feat(pencil): add 'graphiteWidth' option
+
+Fix #42
+```
+
+```commit
+perf(pencil)!: remove graphiteWidth option
+
+BREAKING CHANGE: The graphiteWidth option has been removed.
+
+The default graphite width of 10mm is always used for performance reasons.
+```
+
+## Working with the code
+
+### Set up the workspace
+
+[Fork](https://guides.github.com/activities/forking/#fork) the project, [clone](https://guides.github.com/activities/forking/#clone) your fork, configure the remotes and install the dependencies:
+
+```bash
+# Clone your fork of the repo into the current directory
+$ git clone git@github.com:$(gh config get user -h github.com)/backstage-plugins.git
+# Navigate to the newly cloned directory
+$ cd backstage-plugins
+# Assign the original repo to a remote called "upstream"
+$ git remote add upstream git@github.com:tumido/backstage-plugins.git
+# Install the dependencies
+$ yarn install
+```
+
+The Backstage has multiple dependencies. To pass through their installation, make sure to follow the [isolated-vm](https://github.com/laverdet/isolated-vm#requirements) requirements.
+
+### Lint
+
+The [backstage-plugins](https://github.com/janus-idp/backstage-plugins) repository use [ESLint](https://eslint.org/) for linting and [Prettier](https://prettier.io) for formatting.
+
+Formatting will be automatically verified and fixed by `lint-staged` on the commit.
+
+Before pushing your code changes make sure there are no linting errors with `yarn lint` and `yarn prettier`.
+
+**Tips**:
+
+- Most linting errors can be automatically fixed with `yarn prettier:fix`.
+
+### Tests
+
+The [backstage-plugins](https://github.com/janus-idp/backstage-plugins) repository uses [Jest](https://jestjs.io/) for writing and running tests.
+
+Before pushing your code changes make sure all **tests pass** and the **coverage is 100%**:
+
+```bash
+$ yarn test
+```
+
+## Releasing changes
+
+This repository defaults to a rapid release scheme where we rather release fast, on every PR merge than restrict ourselves by a strict release cadence and policy. This brings contributors the opportunity to see the direct impact of their contributions since they are released immediately after the merge. The release process itself is done via the [semantic-release](https://semantic-release.gitbook.io/semantic-release/) tool. In order for it to work properly, it requires contributors to follow a simple set of rules:
```

Comment:
Missing word in the first sentence: "...where we **would** rather..."

Reply:
Great suggestion, thanks. This is better. 🙂